app/Controller/PagesController.php

```diff
@@ -72,5 +118,59 @@ public function display() {
 			}
 			throw new NotFoundException();
 		}
-	}
+  }
+
+  private $顧客名;
+
+  function set顧客名($v)
+  {
+    $this->顧客名 = $v;
+    return $this;
+  }
+
+  function get顧客名()
+  {
+    return $this->顧客名;
+  }
+}
+
+/**
+ * CyclomaticComplexity
+ */
+// Cyclomatic Complexity = 12
+class Foo {
+    public function example()  {
+        if ($a == $b)  {
+            if ($a1 == $b1) {
+                fiddle();
+            } else if ($a2 == $b2) {
+                fiddle();
+            }  else {
+                fiddle();
+            }
+        } else if ($c == $d) {
+            while ($c == $d) {
+                fiddle();
+            }
+         } else if ($e == $f) {
+            for ($n = 0; $n < $h; $n++) {
+                fiddle();
+            }
+        } else{
+            switch ($z) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Avoid variables with short names like $z. Configured minimum length is 3.](https://www.codacy.com/app/actcat/sideci_php_sandbox/file/1229144208/issues/source?bid=2928887&fileBranchId=2932683#l160)